MAUI/ImageEditor/accessibility.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here about performing actions using accessibility in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+## Accessibility in .NET MAUI Image Editor (SfImageEditor)
+
```

Comment:
**remove**
The [.NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) enables users to interact in different ways to help meet their visual, hearing, speech, dexterity, and cognitive needs.

Reply:
Removed